pkg/doc/verifiable/credential.go

```diff
@@ -426,6 +455,41 @@ func WithCredentialSchemaLoader(loader *CredentialSchemaLoader) CredentialOpt {
 	}
 }
 
+// WithJSONLDValidation uses the JSON LD parser for validation.
+func WithJSONLDValidation() CredentialOpt {
+	return func(opts *credentialOpts) {
+		opts.modelValidationMode = jsonldValidation
+	}
+}
+
+// WithBaseContextAndType validates that only the fields and values (when applicable) are present
+// in the document. No extra fields are allowed (outside of credentialSubject).
+func WithBaseContextAndType() CredentialOpt {
```

Comment:
Let's remove "AndType". Keep things succinct.

(Also elsewhere in this PR).

Reply:
Yeah... I just forgot to change the name here. Fixed.